docs/install.md

```diff
@@ -191,6 +201,135 @@ To do a debug build, run make like:
+options, specifically `--cap-add SYS_ADMIN --security-opt seccomp=unconfined`, are needed so that
+Sandstorm can do its own containerization of itself and of apps within Sandstorm.
+
+This process uses Sandstorm's install.sh to download Sandstorm, and Sandstorm is configured via
+`sandstorm.conf` within the container. To configure and manage the container, note the following.
+
+- Sandstorm manages its own automatic updates in the `/opt/sandstorm` directory, which is
+  inconsistent with the typical Docker approach of using Docker images to manage updates and
+  versioning for application code.
+
+- The install script uses `local.sandstorm.io` and enables development accounts in the `-d` mode. To
+  configure Sandstorm for production use, clear the Docker volume (or create a new one), then remove
+  `-d` from the first `docker run` invocation so that the Sandstorm install script can ask you
+  questions.
+
+- Sandstorm depends on `curl`, `xz`, `openssl`, and `id` from the underlying container. We chose the
+  `buildpack-deps` image because it contains those utilities and is maintained by the Docker
+  team. We recommend periodically updating the `buildpack-deps` container for security reasons and
+  stopping & starting Sandstorm on the new `buildpacks-deps` container.
+
+- Sandstorm doesn't currently update its configuration when run under different environment
+  variables; instead, one must edit the `sandstorm.conf` file within the data volume.
+
+- Sandstorm is a single-machine program, and so you cannot safely run multiple instances of it
+  behind a load balancer on multiple nodes at once. Sandstorm's database lives within the same
+  container and its design currently assumes a single machine.
+
+We're hopeful that the above approach is useful, although we know that it is not the most idiomatic
+use of Docker. If your organization needs deeper integration with Docker, such as a Docker image
+maintained by the Sandstorm team that contains the latest version of Sandstorm, please send an email
+to support@sandstorm.io so we know that customers have a real need for it and can prioritize it
+accordingly.
+
+## Option 7: Use Vagrant when deploying on Mac or Windows
+
+If your organization's servers run Mac OS or Windows, you would need a virtualization tool to run
+Sandstorm, since Sandstorm requires the Linux kernel.
+
+One option is to use Vagrant and VirtualBox; the Sandstorm source repository contains a
+"Vagrantfile" that creates a Linux virtual machine containing Sandstorm. Through the Vagrantfile,
+your Linux virtual machine runs the latest version of Sandstorm, with automatic updates enabled; it
+uses the same install script as described earlier in this document.
+
+You must [install Vagrant](https://www.vagrantup.com/docs/installation/) and a virtualization
+software package such as [VirtualBox](https://www.virtualbox.org/wiki/Downloads), which has the
+upside that it is available free of cost.
```

Comment:
The phrasing here is a little odd - I might just say "You must install Vagrant and a vitualization software package such as VirtualBox.  Both are available free of cost.